fulp_modules/main_features/bloodsuckers/code/bloodsucker/bloodsucker_flaws.dm

```diff
@@ -89,15 +89,15 @@
 		Tremere - Burn in the Chapel, Vassal Mutilation.<br> \
 		Ventrue - Cant drink from mindless mobs, can't level up, raise a vassal instead.<br></span>")
 	if(!isbeefman(bloodsucker))
```

Comment:
now that beefmen can actually benefit from being a part of the malkavian clan, can we let beefmen be malkavians?

if you need their clan choice to have more of an effect, you can make being a malkavian beefman cause a double negative and remove your beefman traumas instead of giving you them (but also still give you x-ray).

Reply:
Beefmen are already insane, they don't need to be part of Malkavian, plus Malkavians get brain trauma that Beefmen cant hide with Masquerade like everyone else